src/solver/batchnorm/backward_spatial_multiple.cpp

```diff
@@ -38,9 +38,34 @@ namespace solver {
 
 namespace batchnorm {
 
+bool BNBwdIsCaseVariant2(const miopen::batchnorm::ProblemDescription& problem)
+{
+    int n, c, h, w;
+    std::tie(n, c, h, w) = tien<4>(problem.GetXDesc().GetLengths());
+
+    unsigned int in_cstride = h * w;
```

Comment:
But actually you can avoid all the prevois `int`-related comment by the following code:
```suggestion
   auto [n, c, h, w] = tien<4>(problem.GetXDesc().GetLengths());
   auto [n_stride, c_stride, h_stride, w_stride] = tien<4>(problem.GetXDesc().GetStrides());
```